lib/models/lookup.js

```diff
@@ -34,13 +34,17 @@ function LookupModelFactory (Model, assert, Errors, Promise, Constants) {
             }
         },
         findByTerm: function (term) {
-            return this.find({
-                or: [
-                    { node: term },
-                    { macAddress: term },
-                    { ipAddress: term }
-                ]
-            });
+            var query = {}; //empty query will find all entries
```

Comment:
I prefer to keep the behavior same with waterline, for many built-in function of waterline, it will tend to handle all records if the query is empty.
In on-http, sometimes we use the waterline built-in function, sometimes use the function that we customized for each data model,  so a consistent behavior will be better.